### DISCUSSION TITLE
Cannot create a new assignment in Github classroom

### DISCUSSION THREAD

**Discussion by okarpenko** [community/community#111231] [GitHub Education]
### Hi everyone,

When I try to create a new assignment in Github classroom, I get "Starter code repo full name not found, please check it again" error message on the first screen, BEFORE I even have a chance to provide the link to the repo with the starter code.
I used Github classroom to create assignments for many years, but suddenly it stopped working.

**Comment by joanh** [comment]
Hi,

Same problem.

**Quick fix**: sign out of `GitHub Classroom` and enter again. It worked for me.

**Reply by okarpenko** [reply]
Thank you, it worked.

**Reply by joanh** [reply]
You're welcome 😇
